app/services/news_feed.rb

```diff
@@ -65,8 +66,13 @@ def fetch(page=nil)
   # Regenerate the user's feed from scratch.
   def regenerate_feed!
     user_set = active_followed_users + [@user.id]
+    group_set = active_joined_groups.to_a
```

Comment:
It was throwing a PG error without the conversion, Can't recall the error exactly.